index.html

```diff
@@ -68,12 +67,12 @@
   </section>
   <section>
     <h2>Introduction</h2>
-    <p>**Performance is a feature.** Modern browsers provide extensive tooling
-    to help developers understand and optimize their applications:
-    network-layer insights about the requests (timing, cache, request and
-    response payload, etc); rendering insights (cost of layout and style
-    recalculations, image decoding, per-layer rendering costs, idle time, etc);
-    CPU and memory profiling; JavaScript debugging, and much more. As a
+    <p><strong>Performance is a feature.</strong> Modern browsers provide
```

Comment:
Was inline markdown supported and removed?

Reply:
“It’s complicated” - but short answer is yes.

Reply:
That's the kind of breaking change that would make pointing to a fixed version and only upgrade after testing valuable.